### PR TITLE
Add aswer solo project enumerated

### PR DESCRIPTION
**Pull request by digomes87**

### CODE REVIEW

File: enumerated.rb [hunk 1]
```diff
@@ -0,0 +1,135 @@
+module Enumerable
+	def my_each
+		arr = self
+		arr.length.times do |x|
+			yield self[x]
+		end
+	end
```
Comment (maelfosso) [inline enumerated.rb]: Should return an **enumerable**

```ruby
irb(main):006:0> array.each
=> #<Enumerator: [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]:each>
irb(main):007:0> array.my_each
Traceback (most recent call last):
        5: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        4: from (irb):7
        3: from enumerated.rb:4:in `my_each'
        2: from enumerated.rb:4:in `times'
        1: from enumerated.rb:5:in `block in my_each'
LocalJumpError (no block given (yield))
irb(main):008:0> array
=> [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]
irb(main):009:0> 
```

File: enumerated.rb [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+	def my_each_with_index
+		arr = self
+		arr.length.times do |x|
+			yield(arr[x], x)
+		end
+	end
```
Comment (maelfosso) [inline enumerated.rb]: Should return an **enumerator**

```ruby
irb(main):008:0> array
=> [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]
irb(main):009:0> array.each_with_index
=> #<Enumerator: [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]:each_with_index>
irb(main):010:0> array.my_each_with_index
Traceback (most recent call last):
        5: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        4: from (irb):10
        3: from enumerated.rb:11:in `my_each_with_index'
        2: from enumerated.rb:11:in `times'
        1: from enumerated.rb:12:in `block in my_each_with_index'
LocalJumpError (no block given (yield))
irb(main):011:0> 
```

File: enumerated.rb [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+	def my_select
+		res = []
+		self.my_each do |x|
+			if yield (x)
+				res.push(x)
+			end
+		end 
+		res
+	end
```
Comment (maelfosso) [inline enumerated.rb]: Should return an **enumerator**

```ruby
irb(main):012:0> array
=> [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]
irb(main):013:0> array.select
=> #<Enumerator: [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]:select>
irb(main):014:0> array.my_select
Traceback (most recent call last):
        7: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        6: from (irb):14
        5: from enumerated.rb:18:in `my_select'
        4: from enumerated.rb:4:in `my_each'
        3: from enumerated.rb:4:in `times'
        2: from enumerated.rb:5:in `block in my_each'
        1: from enumerated.rb:19:in `block in my_select'
LocalJumpError (no block given (yield))
irb(main):015:0> 
```

File: enumerated.rb [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+		res = []
+		self.length.times do |x|
+			res.push(yield(self[x]))
+		end
+		if res.include? false
+			false
+		else
+			true
+		end
+	end
```
Comment (maelfosso) [inline enumerated.rb]:
```ruby
irb(main):017:0> true_array = [1, true, 'hi', []]
=> [1, true, "hi", []]
irb(main):018:0> true_array.all?
=> true
irb(main):019:0> true_array.my_all?
Traceback (most recent call last):
        5: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        4: from (irb):19
        3: from enumerated.rb:28:in `my_all?'
        2: from enumerated.rb:28:in `times'
        1: from enumerated.rb:29:in `block in my_all?'
LocalJumpError (no block given (yield))
irb(main):020:0>
```

Comment (maelfosso) [reply enumerated.rb]:
```ruby
irb(main):022:0> array
=> [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]
irb(main):023:0> array.all?(Integer)
=> true
irb(main):024:0> array.my_all?(Integer)
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):24
        1: from enumerated.rb:26:in `my_all?'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):025:0>
```

Comment (maelfosso) [reply enumerated.rb]:
```ruby
irb(main):027:0> words = %w[dog door rod blade]
=> ["dog", "door", "rod", "blade"]
irb(main):028:0> words.all?(/d/)
=> true
irb(main):029:0> words.my_all?(/d/)
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):29
        1: from enumerated.rb:26:in `my_all?'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):030:0> 
```

Comment (maelfosso) [reply enumerated.rb]:
```ruby
irb(main):032:0> array
=> [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]
irb(main):033:0> array.all?(3)
=> false
irb(main):034:0> array.my_all?(3)
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):34
        1: from enumerated.rb:26:in `my_all?'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):035:0> 
```

File: enumerated.rb [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+	def my_any?
+		self.length.times do |x|
+			a = yield(self[x])
+			if a==true
+				return true
+			end
+		end
+		false
+	end
```
Comment (maelfosso) [inline enumerated.rb]:
```ruby
irb(main):037:0> true_array = [nil, false, true, []]
=> [nil, false, true, []]
irb(main):038:0> true_array.any?
=> true
irb(main):039:0> true_array.my_any?
Traceback (most recent call last):
        5: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        4: from (irb):39
        3: from enumerated.rb:39:in `my_any?'
        2: from enumerated.rb:39:in `times'
        1: from enumerated.rb:40:in `block in my_any?'
LocalJumpError (no block given (yield))
irb(main):040:0> 
```

Comment (maelfosso) [reply enumerated.rb]:
```ruby
irb(main):042:0> array
=> [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]
irb(main):043:0> array.any?(Integer)
=> true
irb(main):044:0> array.my_any?(Integer)
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):44
        1: from enumerated.rb:38:in `my_any?'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):045:0>
```

Comment (maelfosso) [reply enumerated.rb]:
```ruby
irb(main):047:0> words
=> ["dog", "door", "rod", "blade"]
irb(main):048:0> words.any?(/z/)
=> false
irb(main):049:0> words.my_any?(/z/)
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):49
        1: from enumerated.rb:38:in `my_any?'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):050:0>
```

Comment (maelfosso) [reply enumerated.rb]:
```ruby
irb(main):052:0> words
=> ["dog", "door", "rod", "blade"]
irb(main):053:0> words[0] = 'cat'
=> "cat"
irb(main):054:0> words.any?('cat')
=> true
irb(main):055:0> words.my_any?('cat')
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):55
        1: from enumerated.rb:38:in `my_any?'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):056:0> 
```

File: enumerated.rb [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+	def my_none?
+		self.length.times do |x|
+			a = yield(self[x])
+			if a==true
+				return false
+			end
+		end
+		true
+	end
```
Comment (maelfosso) [inline enumerated.rb]:
```ruby
irb(main):058:0> false_array = [nil, false, nil, false]
=> [nil, false, nil, false]
irb(main):059:0> false_array.none?
=> true
irb(main):060:0> false_array.my_none?
Traceback (most recent call last):
        5: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        4: from (irb):60
        3: from enumerated.rb:49:in `my_none?'
        2: from enumerated.rb:49:in `times'
        1: from enumerated.rb:50:in `block in my_none?'
LocalJumpError (no block given (yield))
irb(main):061:0>
```

Comment (maelfosso) [reply enumerated.rb]:
```ruby
irb(main):063:0> array.none?(String)
=> true
irb(main):064:0> array.my_none?(String)
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):64
        1: from enumerated.rb:48:in `my_none?'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):065:0> array
=> [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]
irb(main):066:0> 
```

Comment (maelfosso) [reply enumerated.rb]:
```ruby
irb(main):068:0> words
=> ["cat", "door", "rod", "blade"]
irb(main):069:0> words.none?(/z/)
=> true
irb(main):070:0> words.my_none?(/z/)
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):70
        1: from enumerated.rb:48:in `my_none?'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):071:0> 
```

Comment (maelfosso) [reply enumerated.rb]:
```ruby
irb(main):073:0> words[0] = 5
=> 5
irb(main):074:0> words.none?(5)
=> false
irb(main):075:0> words.my_none?(5)
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):75
        1: from enumerated.rb:48:in `my_none?'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):076:0> words
=> [5, "door", "rod", "blade"]
irb(main):077:0> 
```

File: enumerated.rb [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+				end
+			end
+		elsif arg.length == 0
+			count = self.length
+		else
+			return "You fucked up dickface - 0 or 1 arguments, or a block"
+		end
+
+		count
+	end
```
Comment (maelfosso) [inline enumerated.rb]:
```ruby
irb(main):079:0> array
=> [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]
irb(main):080:0> array.count(3)
=> 13
irb(main):081:0> array.my_count(3)
argument = [3]
=> 0
irb(main):082:0> 
```

Comment (maelfosso) [reply enumerated.rb]: Should return an **enumerator**

```ruby
irb(main):084:0> array
=> [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]
irb(main):085:0> array.map
=> #<Enumerator: [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]:map>
irb(main):086:0> array.my_map
false
Traceback (most recent call last):
        5: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        4: from (irb):86
        3: from enumerated.rb:87:in `my_map'
        2: from enumerated.rb:87:in `times'
        1: from enumerated.rb:88:in `block in my_map'
LocalJumpError (no block given (yield))
irb(main):087:0>
```

File: enumerated.rb [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+	def my_map
+		puts block_given?
+		res = []
+		self.length.times do |x|
+			res.push(yield(self[x]))
+		end
+		res
+	end
```
Comment (maelfosso) [inline enumerated.rb]: Should return an **enumerator**

```ruby
irb(main):084:0> array
=> [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]
irb(main):085:0> array.map
=> #<Enumerator: [5, 5, 4, 1, 4, 1, 5, 7, 4, 1, 0, 3, 0, 4, 4, 3, 3, 0, 0, 8, 5, 8, 4, 2, 1, 8, 5, 7, 7, 7, 8, 7, 5, 5, 7, 0, 8, 6, 0, 3, 7, 5, 8, 2, 6, 5, 8, 2, 4, 3, 4, 4, 3, 5, 6, 4, 1, 4, 6, 3, 7, 1, 7, 7, 7, 5, 3, 4, 4, 6, 3, 5, 4, 5, 5, 2, 3, 2, 6, 2, 0, 1, 5, 2, 5, 6, 3, 2, 1, 6, 6, 1, 6, 3, 3, 4, 7, 4, 4, 1]:map>
irb(main):086:0> array.my_map
false
Traceback (most recent call last):
        5: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        4: from (irb):86
        3: from enumerated.rb:87:in `my_map'
        2: from enumerated.rb:87:in `times'
        1: from enumerated.rb:88:in `block in my_map'
LocalJumpError (no block given (yield))
irb(main):087:0>
```

File: enumerated.rb [hunk 9]
```diff
@@ -0,0 +1,140 @@
+module Enumerable
+
+  def my_each
+    for i in self
+      yield i
+    end
+  end
```
Comment (stratospherique) [inline enumerated.rb]: * [ ] my_each returns an Enumerator if no block is given

File: enumerated.rb [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+
+  def my_each_with_index
+    i = 0
+    for j in self
+      yield j, i
+      i += 1
+    end
+  end
```
Comment (stratospherique) [inline enumerated.rb]: * [ ] my_each_with_index returns an enumerator if no block is given

File: enumerated.rb [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+
+  def my_select
+    arry = []
+    for i in self
+      arry << i if yield i
+    end
+    arry
+  end
```
Comment (stratospherique) [inline enumerated.rb]: * [ ] my_select returns an enumerator if no block is given

File: enumerated.rb [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+          else
+            return false
+          end
+        end
+      else
+        return self.my_all?{|x| x}
+      end
+    end
+    return true
+  end
```
Comment (stratospherique) [inline enumerated.rb]: * [ ] my_all? when a pattern other than Regex or a Class is given returns true if all of the collection matches the pattern

File: enumerated.rb [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+          else
+            next
+          end
+        end
+      else
+        return self.my_any?{|x| x}
+      end
+    end
+    return false
+  end
```
Comment (stratospherique) [inline enumerated.rb]: * [ ] my_any? when a pattern other than Regex or a Class is given returns false if none of the collection matches the pattern

File: enumerated.rb [hunk 14]
```diff
@@ -0,0 +1,140 @@
...
+
+  def my_none?(pattern = nil)
+    if block_given?
+      !self.my_any?{|x| yield x}
+    else
+      !self.my_any?(pattern)
+    end
+  end
```
Comment (stratospherique) [inline enumerated.rb]: * [ ] my_none? when a pattern other than Regex or a Class is given returns true only if none of the collection matches the pattern

File: enumerated.rb [hunk 15]
```diff
@@ -0,0 +1,140 @@
...
+        arry << element
+      end
+    else
+      for i in self
+        element = yield i
+        arry << element
+      end
+    end
+    arry
+  end
```
Comment (stratospherique) [inline enumerated.rb]: * [ ] my_map returns an Enumerator if no block is given

File: enumerated.rb [hunk 16]
```diff
@@ -0,0 +1,140 @@
...
+    for i in self
+      if initialize_num
+        sum = i
+        initialize_num = false
+      else
+        sum = yield sum, i  
+      end
+    end
+    sum
+  end
```
Comment (stratospherique) [inline enumerated.rb]: * [ ] my_inject when a symbol is specified combines each element of the collection by applying the symbol as a named method

File: enumerated.rb (hunk 9 above)
Comment (digomes87) [reply enumerated.rb]: I don't understood very well but fix.

File: .stickler.yml [hunk 17]
```diff
@@ -0,0 +1,11 @@
...
+linters:
+    flake8:
+    eslint:
+        config: './eslint.config'
+    csslint:
+
+files:
+    ignore:
+        - 'src/generated/*'
+        - 'webroot/bower_components/*'
```
Comment (wathigo) [inline .stickler.yml]: **[REQUIRED]** Please include the correct linter configurations. Check [this](https://github.com/microverseinc/linters-config/tree/master/ruby) for the correct configs for Ruby projects. Remember to activate you repository [here](https://stickler-ci.com) after implementing the right configurations.

Comment (digomes87) [reply .stickler.yml]: hey dude, help me in this fuck linter, this shit not function in nothing !!!

**Pull request closed without merging**